lib/components/text_box_component.dart

```diff
@@ -132,7 +132,9 @@ class TextBoxComponent extends PositionComponent with Resizable {
       return;
     }
     prepareCanvas(c);
-    c.drawImage(_cache, Offset.zero, BasicPalette.white.paint);
+    c.drawImage(_cache,
+        Offset.zero,
+        BasicPalette.white.paint..filterQuality = FilterQuality.high);
```

Comment:
This is an amazing finding! Thanks for this :)

Two points:
1. can we format it like so?
```dart
    c.drawImage(
        arg1,
        arg2,
        arg3,
    );
```
2. I wonder if this is also required in `text_config` and `text_component`? I looks like it's only here because it's rendering a pre-rendered image, or is it good to add everywhere?

Reply:
Also, maybe this should be a parameter? Even if with a default value

Reply:
1. yup
2. from the docs, `filterQuality` effects image rendering, so I don't think it matters in any other scenario. I think generally it's good to add if it doesn't hurt performance, which in this case shouldn't be a problem since it's only redrawing on some frames and the image itself is small / not complicated.

I think ideally it should be possible to refactor `TextBoxComponent` to not need to cache an `Image`. That way there would be no quality loss. Is this caching just happening to avoid all the object allocations in each loop? If so, you could just cache `TextPainter` and use it on every loop.

Reply:
@erickzanardo Maybe, but I don't know when you'd want to draw text as a low quality image, I don't think this will have a big performance impact (I haven't done extensive testing though)

Reply:
@kurtome makes sense, we can roll out this as is for me :)

the cache is intended to increase performance, specially once the text is all drawn, to avoid computing everything